packages/cozy-harvest-lib/src/components/AccountForm.jsx

```diff
@@ -36,6 +36,9 @@ export class AccountField extends PureComponent {
       className: 'u-m-0', // 0 margin
       fullwidth: true,
       label: t(`fields.${localeKey}.label`, { _: name }),
+      placeholder: placeholder
+        ? placeholder
+        : t(`fields.${localeKey}.placeholder`, { _: '' }),
```

Comment:
For placeholders, we do not want to use predefined labels so we should use instead `t(\`fields.${name}.placeholder`, { _: '' })